Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp

```diff
@@ -134,15 +134,43 @@ static WebFrameMap& webFrameMap()
     return map;
 }
 
-static WebKitFrame* webkitFrameGetOrCreate(WebFrame* webFrame)
+static WebKitFrame* webkitFrameGet(WebFrame* webFrame)
 {
     auto wrapperPtr = webFrameMap().get(webFrame);
     if (wrapperPtr)
         return wrapperPtr->webkitFrame();
 
+    return nullptr;
+}
+
+static CString getDocumentLoaderURL(DocumentLoader* documentLoader)
+{
+    if (!documentLoader)
+        return { };
```

Comment:
Can it be nullptr? I have an assert in current code.

Reply:
After move setting uri to WebKitWebFrame initialization I put assert back here